### ISSUE TITLE
Error with new appimages.

### ISSUE THREAD

**Issue by ebarranco** [synfig/synfig#548]
**Synfig version & platform**:
SynfigStudio-1.3.7-testing-18.05.07-linux64-7ad82.appimage
Manjaro Linux KDE (17.1.10)

**Issue description**:

The file doesn't execute.
 
Command line error:

/tmp/.mount_d2SCEE/usr/bin/synfigstudio: symbol lookup error: /usr/lib/libfontconfig.so.1: undefined symbol: FT_Done_MM_Var

**Comment by morevnaproject** [comment]
Hmmm... That's weird - I do not remember we changed anything in our build configuration. Can you please check if old version (1.3.6) works for you? - https://sourceforge.net/projects/synfig/files/snapshots/1.3.6-20180126/

**Comment by morevnaproject** [comment]
Please let me know your version of fontconfig.

@blackwarthog SHould we consider bundling fontconfig into appimage?

**Comment by ebarranco** [comment]
Name            : fontconfig
Version         : 2.13.0+10+g58f5285-1
Architecture    : x86_64

**Comment by ice0** [comment]
@morevnaproject checked on Linux Mint fontconfig 2.11.94 - works fine.
@ebarranco did you have libfontconfig installed?

**Comment by ebarranco** [comment]
No, don't have libfontconfig. @ice0

**Comment by ice0** [comment]
@ebarranco sorry for long waiting. As i see here (https://www.archlinux.org/packages/extra/x86_64/fontconfig/files/) in Manjaro libfontconfig is already included in fontconfig. But that symbol 'FT_Done_MM_Var'... FT is looks like FreeType dependency. Which version of FreeType do you have installed?

**Comment by ice0** [comment]
I think i got it. Manjaro have the newest Freetype2 library, in which they broken backward compatibility. As i see here (https://github.com/probonopd/linuxdeployqt/issues/259) they just  remove ./lib/libfreetype.so.6 from AppImage. 
@morevnaproject @blackwarthog 
I can't check this because AppImage is not unpacking using '--appimage-unpack' switch (other AppImage which i have is working fine).

**Comment by Translator5** [comment]
Approved - Cant start Synfig installed by AUR as well as the latest appimages caused by same issue.
System: Манџаро Linux as well

**Comment by ice0** [comment]
Hello guys!

TLDR
This issue is fixed and updated version of Synfig AppImage will be uploaded ASAP. 

Here some details. 
We discussed this error with @blackwarthog and what is how problem looks like for now:
- FreeType 2.9 added a new function  'FT_Done_MM_Var' (https://sourceforge.net/projects/freetype/files/freetype2/2.9/)
- Fontconfig uses Freetype
- Synfig appimage includes older version of Freetype, but not Fontconfig (where was a strange error with it earlier, so it was removed)

First i was think this is newer fontconfig 2.13 issue, but my Fedora 28 installation also has fontconfig 2.13 and Synfig AppImage work fine. But after some digging i realized what Fedora have freetype 2.8, where Manjaro have freetype 2.9, so here may be different compile flags or patches from Fedora for freetype.
Next @blackwarthog returned fontconfig to AppImage and did a test build which seems to run fine.

You can download testing version here: https://dev.icystar.com/downloads/SynfigStudio-1.3.8-testing-18.05.25-linux64-89479.appimage

**Comment by ebarranco** [comment]
@ice0 Yeah! It is working now. Thanks a lot.

**Comment by morevnaproject** [comment]
Shipped with version 1.3.9 - https://www.synfig.org/2018/06/25/synfig-studio-1-3-9-released/
